tests/spec/shell/cat.spec.js

```diff
@@ -1,20 +1,25 @@
 var Filer = require('../../..');
 var util = require('../../lib/test-utils.js');
 var expect = require('chai').expect;
+var optionals = require('../../../src/shell/optional');
 
-describe('FileSystemShell.cat', function() {
+describe('(optional) FileSystemShell.cat', function() {
```

Comment:
Why mark these as optional? I'm don't see how that improves the test reporting, and it may be confusing.